addons/advanced_ballistics/stringtable.xml

```diff
@@ -48,6 +50,7 @@
             <German>Aktiviert die erweiterte Ballistik</German>
             <Czech>Aktivuje pokročilou balistiku</Czech>
             <Portuguese>Ativa balística avançada</Portuguese>
+            <Hungarian>Engedélyezi a fejlett ballisztikát</Hungarian>
```

Comment:
They're contextually close to make sense either way, but I might adjust it later on. "Enabling" would mean the same as "Activating" here.